generateSecrets.js

```
@@ -0,0 +1,33 @@
+  } else {
+    envContent += `\n${key}=${value}`;
+  }
+  return envContent;
+}
+
+envContent = updateEnvContent(envContent, 'JWT_ACCESS_SECRET', accessSecret);
+envContent = updateEnvContent(envContent, 'JWT_REFRESH_SECRET', refreshSecret);
+envContent = updateEnvContent(envContent, 'JWT_RESET_SECRET', resetSecret);
+envContent = updateEnvContent(envContent, 'JWT_CONFIRMATION_SECRET', confirmationSecret);
+
+fs.writeFileSync(envFilePath, envContent, 'utf8');
```

ESLint

[error] Extra semicolon.

(semi)